src/features/favorites/components/NotConnectedFavorites.tsx

```diff
@@ -1,94 +1,56 @@
+export const NotConnectedFavorites = () => (
+  <GenericInfoPageWhite
+    title="Identifie-toi pour retrouver tes favoris"
+    subtitle="Ton compte te permettra de retrouver tous tes bons plans en un clin d’oeil&nbsp;!"
+    titleComponent={StyledTitle4}
+    subtitleComponent={CenteredText}
+    icon={BicolorUserFavorite}
+    separateIconFromTitle={false}>
+    <Row>
+      <ButtonContainer>
+        <InternalTouchableLink
+          as={ButtonWithLinearGradient}
+          wording="Créer un compte"
+          navigateTo={{ screen: 'SignupForm' }}
+          onBeforeNavigate={analytics.logSignUpFromFavorite}
+          buttonHeight="tall"
+        />
         <Spacer.Column numberOfSpaces={4} />
-
-        <TextContainer>
-          <CenteredText>
-            <StyledBody>
-              Ton compte te permettra de retrouver tous tes favoris en un clin d’oeil&nbsp;!
-            </StyledBody>
-          </CenteredText>
-        </TextContainer>
-      </CenteredContainer>
-
-      <Row>
-        <ButtonContainer>
-          <InternalTouchableLink
-            as={ButtonPrimaryWhite}
-            wording="S’inscrire"
-            navigateTo={{ screen: 'SignupForm' }}
-            onBeforeNavigate={analytics.logSignUpFromFavorite}
-            buttonHeight="tall"
-          />
-          <Spacer.Column numberOfSpaces={4} />
-          <InternalTouchableLink
-            as={ButtonTertiaryWhite}
-            wording="Se connecter"
-            navigateTo={{ screen: 'Login' }}
-            onBeforeNavigate={analytics.logSignInFromFavorite}
-            buttonHeight="tall"
-          />
-        </ButtonContainer>
-      </Row>
-      <Spacer.Column numberOfSpaces={12} />
-      <Spacer.Flex />
-      <Spacer.BottomScreen />
-    </Container>
-  )
-}
-
-const StyledUserFavorite = styled(BicolorUserFavorite).attrs(({ theme }) => ({
-  color: theme.colors.white,
-  color2: theme.colors.white,
-  size: theme.illustrations.sizes.fullPage,
-}))``
-
-const Container = styled.View({
-  flex: 1,
-  alignItems: 'center',
-})
+        <StyledAuthenticationButton
+          type="login"
+          onAdditionalPress={analytics.logSignInFromFavorite}
+        />
+      </ButtonContainer>
+    </Row>
+    <Spacer.Column numberOfSpaces={12} />
+    <Spacer.BottomScreen />
+  </GenericInfoPageWhite>
+)
 
 const Row = styled.View({ flexDirection: 'row' })
 
-const StyledTitle4 = styled(Typo.Title4)(({ theme }) => ({
-  textAlign: 'center',
-  color: theme.colors.white,
-}))
-
-const CenteredContainer = styled.View({
-  flexGrow: 1,
-  alignItems: 'center',
-  justifyContent: 'center',
+const StyledTitle4 = styled(Typo.Title4)({
   textAlign: 'center',
-  marginHorizontal: getSpacing(8),
+  marginBottom: getSpacing(4),
 })
 
-const ButtonContainer = styled.View({ flex: 1, maxWidth: getSpacing(44) })
-
-const TextContainer = styled.View({ maxWidth: getSpacing(88) })
+const ButtonContainer = styled.View({ flex: 1 })
```

Comment:
Est-ce que tu peux faire :

```suggestion
const ButtonContainer = styled.View({
   flex: 1
  paddingBottom: getSpacing(10),
})
```

et supprimer la ligne 36 : 
````
    <Spacer.Column numberOfSpaces={12} />
`````

car j'ai l'impression que c'est ce qu'on fait dans d'autres pages qui utilisent `GenericInfoPageWhite` comme `AccountReactivationSuccess`, `CulturalSurveyThanks` ou encore `OnboardingGeolocation`